index.js

```diff
@@ -26,6 +29,23 @@ function loadFormatter(formatterPath) {
 }
 
 /**
+ * Creates the output dir
+ * @param {String} filePath
+ * @param cb
+ */
+function createDirectory(filePath, cb) {
+    var dirname = path.dirname(filePath);
+
+    mkdirp(dirname, function (err) {
+        if (!err) {
+            cb();
+        } else {
+            console.error(err);
```

Comment:
I would improve the error messaging here by doing something like `console.error("Error creating directory: ", err)`